third_party/golang/parse/lex.go

```diff
@@ -0,0 +1,262 @@
+package parse
```

Comment:
did you write this?  If so it shouldn't be in third_party.  If you didn't, it should be managed with godeps.